components/Flashcard/Flashcard.js

```diff
@@ -1,30 +1,100 @@
 import styled from "styled-components";
+import { useState } from "react";
 
 export default function Flashcard({ question, answer, collection }) {
+  const [showAnswer, setShowAnswer] = useState(false);
+
+  function handleShowAnswer() {
+    setShowAnswer(!showAnswer);
+  }
+
   return (
-    <StyledFlashcard>
-      <CollectionTitle>{collection}</CollectionTitle>
-      <Question>{question}</Question>
-      <p>{answer}</p>
-    </StyledFlashcard>
+    <CardContainer onClick={handleShowAnswer}>
+      <StyledFlashcard $showAnswer={showAnswer}>
+        <CardFront>
+          <CollectionTitle>{collection}</CollectionTitle>
+          <Question>{question}</Question>
+          <StyledArrow
+            xmlns="http://www.w3.org/2000/svg"
+            height="24px"
+            viewBox="0 -960 960 960"
+            width="24px"
+            fill="#000"
+          >
+            <path d="m700-300-57-56 84-84H120v-80h607l-83-84 57-56 179 180-180 180Z" />
+          </StyledArrow>
+        </CardFront>
+        <CardBack>
+          <Answer>{answer}</Answer>
+          <StyledArrow
+            xmlns="http://www.w3.org/2000/svg"
+            height="24px"
+            viewBox="0 -960 960 960"
+            width="24px"
+            fill="#000"
+            transform="scale(-1 1)"
+          >
+            <path d="m700-300-57-56 84-84H120v-80h607l-83-84 57-56 179 180-180 180Z" />
+          </StyledArrow>
+        </CardBack>
+      </StyledFlashcard>
+    </CardContainer>
   );
 }
 
+const CardContainer = styled.div`
+  perspective: 1000px;
+`;
+
 const StyledFlashcard = styled.section`
+  width: 19rem;
```

Comment:
I think an `article` would be more appropriate here. In your context, each flashcard represents a discrete piece of content that makes sense on its own. It contains all the necessary information (a question, an answer, a collection title) to be understood without needing additional context from the surrounding page.

Reply:
We have also adopted your suggestion here.